includes/specials/SpecialUserlogout.php

```diff
@@ -114,4 +97,15 @@ protected function isValidRedirectUrl( $url ) {
 		$hostname = parse_url( $url, PHP_URL_HOST );
 		return preg_match( '/(\.|^)(wikia|fandom)\.com$/', $hostname );
 	}
+
+	/**
+	 * @param $parsed_url
+	 * @return string hostname url
+	 */
+	protected function getHostname($parsed_url) {
```

Comment:
I'm pretty sure we avoid using snake case in variable names.

Also, don't we already have a "getHostname" method somewhere in PHP / Mediawiki / App PHP codebase?